settings/LoanPolicyDetail.js

```diff
@@ -38,6 +38,12 @@ class LoanPolicyDetail extends React.Component {
     this.cViewMetaData = props.stripes.connect(ViewMetaData);
   }
 
+  translate(id) {
```

Comment:
No need in this func as stated above Same,  \<FormattedMessage\> component could be used instead